src/Nethermind/Nethermind.Baseline.Test/JsonRpc/BaselineModuleTests.cs

```diff
@@ -79,7 +80,7 @@ public async Task deploy_deploys_the_contract()
 
             await testRpc.AddBlock();
 
-            testRpc.BlockTree.Head.Number.Should().Be(5);
+            testRpc.BlockTree.Head.Number.Should().Be(2);
```

Comment:
why would this number change from 5 to 2?

Reply:
I removed additional blocks deployed at start